### ISSUE TITLE
App Runner : Please support AutoScalingConfiguration settings in manifest.yml

### ISSUE THREAD

**Issue by sogaoh** [aws/copilot-cli#3200]
I want to use `min_size` of AutoScalingConfiguration setting.
I think 1 is default, but I want to increase it to 2. 
So, please support AutoScalingConfiguration settings in manifest.yml. 

refs:
- https://docs.aws.amazon.com/apprunner/latest/dg/manage-autoscaling.html
- https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/apprunner_auto_scaling_configuration_version
- https://docs.aws.amazon.com/apprunner/latest/api/API_AutoScalingConfiguration.html
- https://docs.aws.amazon.com/apprunner/latest/api/API_DescribeAutoScalingConfiguration.html

**Comment by efekarakus** [comment]
Hi @sogaoh 👋 

Thank you for the feature request! I think right now the issue is that Copilot is blocked on CloudFormation support:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apprunner-service.html#cfn-apprunner-service-autoscalingconfigurationarn

Today, the autoscaling configuration does not allow an object with `MaxSize`, `MaxConcurrency`... and instead strictly an ARN. Once an object as an input is supported in CloudFormation then Copilot can also surface it in the manifest

**Comment by schm** [comment]
Hey,

I understand that this is blocked by CloudFormation's abilities. But maybe it would be possible to at least expose the `AutoScalingConfigurationArn` configuration? We can then still set up the configurations manually or using other tools. But then we could reference existing configurations for different services. I would assume, that should be possible today and might not be such a big effort?

I would be looking forward to such a feature.

**Comment by iamhopaul123** [comment]
Hello @schm the request makes sense to me. For now we could support the import experience by allowing

```yaml
count: arn:aws:apprunner:us-east-1:123456789012:autoscalingconfiguration/high-availability/3
```

in the manifest.

**Comment by Lou1415926** [comment]
Hi all 👋🏼 !

We just released [v1.24.0](https://github.com/aws/copilot-cli/releases/tag/v1.24.0). You can now provide the name of an autoscaling configuration for your request-driven web services. Check out our [release note](https://github.com/aws/copilot-cli/releases/tag/v1.24.0) for more!

**Comment by sogaoh** [comment]
@Lou1415926 @efekarakus 
Hi. 

There is no description of `count` settings on https://aws.github.io/copilot-cli/ja/docs/manifest/rd-web-service/, when do you plan to add it?

**Comment by hkford** [comment]
Hi @sogaoh 
Currently we are translating Copilot v1.23 & v1.24 documentation into Japanese. Maybe we could add it by the end of this year.

**Comment by hkford** [comment]
Now published [Copilot v1.24 documentation in Japanese](https://aws.github.io/copilot-cli/ja/blogs/release-v124/).
It includes [the `count` setting](https://aws.github.io/copilot-cli/ja/docs/manifest/rd-web-service/#count). It looks rendering of yaml block is broken and I will fix it soon.
